lib/logstash/inputs/twitter.rb

```diff
@@ -46,7 +46,13 @@ class LogStash::Inputs::Twitter < LogStash::Inputs::Base
   config :oauth_token_secret, :validate => :password, :required => true
 
   # Any keywords to track in the twitter stream
-  config :keywords, :validate => :array, :required => true
+  config :keywords, :validate => :array, :required => false
+
+  # Any locations to track in the twitter stream
```

Comment:
As for locations, I think the API takes latitude-longitude bounding boxes (4 numbers in sequence), so we should help users by documenting the expected format with maybe a few examples.